cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/AbstractSchedulerService.java

```diff
@@ -72,8 +72,8 @@ protected final void startScheduler() {
     try {
       timeScheduler.start();
       LOG.info("Started time scheduler");
-    } catch (SchedulerException e) {
-      LOG.error("Error starting time scheduler {}", e.getCause(), e);
+    } catch (org.quartz.SchedulerException e) {
```

Comment:
why was this changed? By the way, should we have cdap SchedulerException that can be thrown by all schedulers? That would mean the time scheduler would have to catch quartz exceptions and wrap them into our SchedulerException. But it would be nice because it decouples the abstract scheduler service from the quartz implementation of time scheduler.